integration/repo_test.go

```diff
@@ -397,6 +397,24 @@ func TestRepository(t *testing.T) {
 				want.ArtifactType = artifact.TypeFilesystem
 			},
 		},
+		{
+			name: "julia generating SPDX SBOM",
+			args: args{
+				command: "rootfs",
+				format:  "spdx-json",
+				input:   "testdata/fixtures/repo/julia",
+			},
+			golden: "testdata/julia-spdx.json.golden",
+		},
+		{
+			name: "julia generating CycloneDX SBOM",
+			args: args{
+				command: "rootfs",
+				format:  "cyclonedx",
+				input:   "testdata/fixtures/repo/julia",
+			},
+			golden: "testdata/julia-cyclonedx.json.golden",
+		},
```

Comment:
We [introduced the intermediate representation for SBOM](https://github.com/aquasecurity/trivy/pull/6240). We don't need tests for SPDX and CycloneDX unless Julia has any specific handling. Either of them is fine.

Reply:
No specific handling is needed. I have removed the CycloneDX test.